app/src/components/market/sections/market_create/market_wizard_creator.tsx

```diff
@@ -55,7 +55,7 @@ export const MarketWizardCreator = (props: Props) => {
 
   const [currentStep, setCurrentStep] = useState(1)
   const [marketData, setMarketdata] = useState<MarketData>(marketDataDefault)
-  const [first, setFirst] = useState<number>(19)
+  const [first, setFirst] = useState<number>(8)
```

Comment:
This should still be set to `useState<number>(19)`, I was just making a note that _eventually_ we will have to change it back.